### ISSUE TITLE
Forgotten password link does not work once you have put your email address in

### ISSUE THREAD

**Issue by rsmith013** [cedadev/datamad2#99]
https://datamad-test.ceda.ac.uk/accounts/password_reset/

**Comment by rsmith013** [comment]
```[ERROR] [2020-03-31 14:23:35,494] [django.request:228] [waitress] Internal Server Error: /accounts/password_reset/
Traceback (most recent call last):
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/lib64/python3.6/contextlib.py", line 52, in inner
    return func(*args, **kwds)
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/utils/decorators.py", line 45, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/utils/decorators.py", line 142, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/contrib/auth/views.py", line 220, in dispatch
    return super().dispatch(*args, **kwargs)
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/views/generic/base.py", line 97, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/views/generic/edit.py", line 142, in post
    return self.form_valid(form)
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/contrib/auth/views.py", line 233, in form_valid
    form.save(**opts)
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/contrib/auth/forms.py", line 291, in save
    for user in self.get_users(email):
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/contrib/auth/forms.py", line 271, in get_users
    'is_active': True,
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/db/models/manager.py", line 82, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/db/models/query.py", line 892, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/db/models/query.py", line 910, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/db/models/sql/query.py", line 1290, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/db/models/sql/query.py", line 1318, in _add_q
    split_subq=split_subq, simple_col=simple_col,
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/db/models/sql/query.py", line 1190, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/db/models/sql/query.py", line 1049, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/home/waitress/venv/lib64/python3.6/site-packages/django/db/models/sql/query.py", line 1420, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'is_active' into field. Choices are: data_centre, email, first_name, groups, id, is_admin, is_superuser, last_login, last_name, logentry, password, user_permissions, username```

**Comment by rsmith013** [comment]
fixed by d730b270982409b56a2ff663e8a68e5bee44b036
